src/Tribe/Utils/Post_Thumbnail.php

```diff
@@ -286,17 +286,22 @@ public function serialize() {
 		$data            = $this->fetch_data();
 		$data['post_id'] = $this->post_id;
 
-		return serialize( $data );
+		return wp_json_encode( $data );
 	}
 
 	/**
 	 * {@inheritDoc}
 	 */
 	public function unserialize( $serialized ) {
-		$data          = unserialize( $serialized );
+		$data          = json_decode( $serialized, true );
+		array_walk( $data, static function ( &$data_entry ) {
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: ac9bd55f85db0d4ee7ab9dc269acef01</sub>